src/modules/poweraccent/PowerAccent.Core/Services/SettingsService.cs

```diff
@@ -16,10 +16,12 @@ public class SettingsService
     private readonly ISettingsUtils _settingsUtils;
     private readonly IFileSystemWatcher _watcher;
     private readonly object _loadingSettingsLock = new object();
+    private PowerToys.PowerAccentKeyboardService.KeyboardListener _keyboardListener;
```

Comment:
can we shorten adding a using directive?

Reply:
done